### DISCUSSION TITLE
Can Shadcn Vue be installed in Vitepress?

### DISCUSSION THREAD

**Discussion by wolono** [unovue/shadcn-vue#785] [Q&A]
I have built a document site using Vitepress and I want to install shadcn ui in Vitepress to refactor the site's themes. If possible, how can we do it? We need everyone's help

**Comment by enzonotario** [comment]
You can install it as any other Vue package. You can see a working example here: https://github.com/enzonotario/vitepress-openapi

Feel free to ask anything.

**Comment by Alaye-Dong** [comment]
I've checked the documentation and manually imported the components, but this is causing 404 errors on pages where I use shadcn-vue components. Has anyone successfully implemented this？
